examples/flax/question-answering/utils_qa.py

```diff
@@ -0,0 +1,427 @@
+# coding=utf-8
```

Comment:
If this is a copy from the PyTorch example folder, it should be added [here](https://github.com/huggingface/transformers/blob/04976a32dc555667afa994e8f918cbee88d84a4f/utils/check_copies.py#L31) so the copy is checked by our internal tooling.

Reply:
Done